README.md

```diff
@@ -232,35 +232,10 @@ the log can be found in the source folder under:`apps/aecore/logs`
 
 **Registering an oracle:**
   `Oracle.register(query_format, response_format, query_fee, fee, ttl)`
-  The query and response formats are treated as json schemas by which the queries and responses
-  are validated. The query fee is the minimum fee that will be required for queries made to the
+  The query and response formats are strings. The query fee is the minimum fee that will be required for queries made to the
```

Comment:
"The query and response formats are string descriptions of what the input for those should be." instead of "The query and response formats are strings."

Reply:
ok, I will change it